app/src/main/java/ar/com/wolox/android/training/ui/login/LogInPresenter.java

```diff
@@ -19,7 +19,9 @@
 
     private RetrofitServices mRetrofitServices;
 
-    // Constructor
+    /*
```

Comment:
Keep in mind that a `javadoc` uses `two` asterisks. Only one is considered a normal block of comments.